packages/discovery/src/discovery/provider/DiscoveryEtherscanLikeClient.ts

```diff
@@ -0,0 +1,15 @@
+import { EtherscanLikeClient, HttpClient, Logger } from '@l2beat/shared'
+import { UnixTime } from '@l2beat/shared-pure'
+
+export class DiscoveryEtherscanLikeClient extends EtherscanLikeClient {
```

Comment:
> On the other hand, we can add a static construction method to wrap minimum timestamp and spare us yet another class. It will simply partially apply rest arguments with fixed timestamp

I like this one more.